website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -546,6 +605,66 @@ Specifies a mode for how the service directs inbound and outbound traffic.
   - `direct`: The proxy's listeners must be dialed directly by the local application and other proxies.
   - `transparent`: The service captures inbound and outbound traffic and redirects it through the proxy. The mode does not enable the traffic redirection. It instructs Consul to configure Envoy as if traffic is already being redirected.
 
+### `RateLimits`
+
+Map containing an instance-level configuration for the service. 
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `RateLimits{}.InstanceLevel`
+
+Map containing a set of request rate limit configurations for instances of the service. 
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `RateLimits{}.InstanceLevel{}.RequestsPerScond`
+
+Specifies the average number of requests per second allowed to the service. Consul allows requests to the service when the average number of requests per second exceeds this value, but it temporarily lowers the speed of the transactions. You must also specify the [`RequestsMaxBurst`](#ratelimits-instancelevel-requestsmaxburst) parameter. The number of requests per second to the service can exceed the limit specified in the `RequestsPerSecond` parameter, but not the `RequestsMaxBurst` parameter. 
```

Comment:
Not strictly required, since RequestsMaxBurst defaults to RequestsPerSecond
```suggestion
Specifies the average number of requests per second allowed to the service. Consul allows requests to the service when the average number of requests per second exceeds this value, but it temporarily lowers the speed of the transactions. You may also specify the [`RequestsMaxBurst`](#ratelimits-instancelevel-requestsmaxburst) parameter. The number of requests per second to the service can exceed the limit specified in the `RequestsPerSecond` parameter, but not the `RequestsMaxBurst` parameter. 
```